Makefile

```diff
@@ -84,6 +84,7 @@ ifeq ($(LOCAL_OS),Linux)
    TARGET_OS_LOCAL = linux
 else ifeq ($(LOCAL_OS),Darwin)
    TARGET_OS_LOCAL = darwin
+   PATH := $(PATH):$(HOME)/go/bin/darwin_$(GOARCH)
```

Comment:
Is this true for everyone? I don't have this folder.

Reply:
Same for me, but this is the case for the Github CI runner. We are only appending the path so there is no harm.